python/hail/dataset.py

```diff
@@ -235,33 +264,22 @@ def annotate_alleles_expr(self, condition, propagate_gq=False):
             pargs.append('--propagate-gq')
         return self.hc.run_command(self, pargs)
 
-    def annotate_global_expr_by_variant(self, condition):
-        """Update the global annotations with expression with aggregation over
-        variants.
+    def annotate_global_py(self, annotation, ptype, path):
```

Comment:
I'd probably put the path first in analog with `va.info.callStats = gs.callStats()`, etc.